assets/src/block-validation/amp-validation-status/status-notification.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button, ExternalLink } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import AMPValidationErrorsKeptIcon from '../../../images/amp-validation-errors-kept.svg';
+import { BLOCK_VALIDATION_STORE_KEY } from '../store';
+import { StatusIcon } from '../icon';
+import { SidebarNotification } from '../../block-editor/components/sidebar-notification';
+
+/**
+ * AMP validation status notification component.
+ */
+export default function AMPValidationStatusNotification() {
+	const { autosave, savePost } = useDispatch( 'core/editor' );
+
+	const {
+		ampCompatibilityBroken,
+		fetchingErrorsRequestErrorMessage,
+		hasValidationErrors,
+		isDraft,
+		isEditedPostNew,
+		isFetchingErrors,
+		reviewLink,
+	} = useSelect( ( select ) => ( {
+		ampCompatibilityBroken: select( BLOCK_VALIDATION_STORE_KEY ).getAMPCompatibilityBroken(),
+		fetchingErrorsRequestErrorMessage: select( BLOCK_VALIDATION_STORE_KEY ).getFetchingErrorsRequestErrorMessage(),
+		hasValidationErrors: select( BLOCK_VALIDATION_STORE_KEY ).getValidationErrors()?.length > 0,
```

Comment:
The `getValidationErrors` selector would always return an array, no?

Reply:
I removed the `?` as part of 8b42ad96e385431f71f561818126ef8d0fcd8cd9